recipe/meta.yaml

```
@@ -78,8 +80,7 @@ test:
 
 about:
   home: https://developer.nvidia.com/nvcomp
-  license:
-    LicenseRef-nvCOMP-Software-License-Agreement AND LicenseRef-NVIDIA-End-User-License-Agreement
+  license: LicenseRef-nvCOMP-Software-License-Agreement AND LicenseRef-NVIDIA-End-User-License-Agreement
```

If we need to wrap it, we could do this

```suggestion
  license: >-
    LicenseRef-nvCOMP-Software-License-Agreement AND LicenseRef-NVIDIA-End-User-License-Agreement
```